tests/hmm/hmumu_utils.py

```diff
@@ -2303,6 +2324,7 @@ def dnn_variables(hrelresolution, leading_muon, subleading_muon, leading_jet, su
 
     nw = len(leading_jet["eta"])
     fixm = NUMPY_LIB.full(nw, 125.0, dtype=NUMPY_LIB.float32)
+    mhfordnn(fixm, mm_sph["mass"])
```

Comment:
something like this would also work, without requiring an explicit loop:

```python
#make a copy to avoid changing the original mass values
fixm = mm_sph["mass"].copy()
fixm[(fixm > 135)] = 125
fixm[(fixm < 115)] = 125
```

Reply:
sounds good!

Reply:
Done.